.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,5 @@
-Add the following to your main PR comment
```

Comment:
this is totally unrelated to the PR but removing the code tags allows this template to autoformat